clj/src/slipstream/ui/models/vms.clj

```diff
@@ -11,7 +11,7 @@
                     :measurement])
       (assoc        :username           (-> vm-metadata :user       not-empty))
       (assoc        :cloud-instance-id  (-> vm-metadata :instanceId not-empty))
-      (assoc        :run-uuid           (-> vm-metadata :runUuid    (s/replace "Unknown" "") not-empty))
```

Comment:
To handle the same way the following two occurrences of an unknown run uuid:

1. the attribute `runUuid` is not present in the metadata of the vm
1. the value of the attribute `runUuid` is `Unknown`

Both as represented by a `nil` in the metadata structure. This offers a [coherent way](https://github.com/slipstream/SlipStreamUI/blob/3bc4a7cdb8a4204c10ffef27b035ae71050e2a52/clj/src/slipstream/ui/views/tables.clj#L743) to handle the display of an unknown run uuid in the UI.